src/analyze.test.ts

```diff
@@ -263,6 +271,34 @@ Object {
     });
   });
 
+  describe('tags', () => {
+    it('template literal tags', () => {
```

Comment:
```suggestion
    it('string literal tags', () => {
```

Reply:
Do you also want to add a test for template literal tags, e.g.

```
tags={[`docs`]}
```

I believe this will currently fail and that is the intended/desired behavior.

Reply:
Yes, I will push shortly

Reply:
I've added the test along with the throw error code